crates/bindings-csharp/Runtime/Internal/Autogen/RawRowLevelSecurityDefV9.cs

```diff
@@ -18,9 +18,7 @@ public partial class RawRowLevelSecurityDefV9
 		[DataMember(Name = "sql")]
 		public string Sql;
 
-		public RawRowLevelSecurityDefV9(
-			string Sql
-		)
+		public RawRowLevelSecurityDefV9(string Sql)
```

Comment:
Do these files get auto-formatted? If so, why did they change?

Reply:
No, I changed the C# codegen logic while unifying reducer, tables and type generation. They're not autoformatted.